pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```

Comment:
Please remove this one from the PR. This should be fixed upstream in node2nix.

Reply:
https://github.com/svanderburg/node2nix/pull/174

Reply:
Thank you! Fixed.